MapboxDirectionsTests/OfflineDirectionsTests.swift

```diff
@@ -24,14 +24,13 @@ class OfflineDirectionsTests: XCTestCase {
             return OHHTTPStubsResponse(jsonObject: jsonObject, statusCode: 200, headers: ["Content-Type": "application/json"])
         }
         
-        directions.fetchAvailableOfflineVersions { (versions, error) in
+        directions.fetchAvailableOfflineVersions(completionHandler: { (versions, error) in
```

Comment:
does trailing closure syntax still work here?

Reply:
yes, it does. I think there was a battle with autocomplete during the refactor. updated.